schema/deploy/warehouse/target/data.sql

```diff
@@ -80,6 +80,8 @@ with target_lineage (identifier, lineage) as (
         ('CoV_229E_CoV_OC43',           'Human_coronavirus'),
         ('CoV_HKU1_CoV_NL63',           'Human_coronavirus'),
         ('COVID-19',                    'Human_coronavirus.2019'),
+        ('COVID-19_Orf1b',              'Human_coronavirus.2019'),
+        ('COVID-19-S_gene',             'Human_coronavirus.2019'),
```

Comment:
Is the plan now to drop this commit because we're deleting the affected records?

Reply:
I think we still want to link them in case for whatever reason they get pushed to ID3C again. That way these results are hidden from public view...

Reply:
Agreed.